src/utils.js

```diff
@@ -260,3 +260,16 @@ export function addMultipleEventListeners(_els, handler, _events) {
     });
   };
 }
+
+export const focusableElements = [
+  'a[href]',
+  'area[href]',
+  'input:not([disabled])',
```

Comment:
Need to also account for `type="hidden"`

Reply:
👍